src/Codeception/Test/Feature/CodeCoverage.php

```diff
@@ -26,8 +26,20 @@ public function codeCoverageEnd($status, $time)
             return;
         }
 
+        if (method_exists($this, 'getLinesToBeCovered')) {
```

Comment:
It may be possible to get something like this working in Cept/Gherkin, but the PHPUnit helper my implementation uses is looking for the `@covers` and `@uses` annotations on a test method (and on the test class, where it also checks for `@coversDefaultClass` and maybe some other stuff I'm not aware of) which pretty much rules out any test format that doesn't use a class-method structure. If I become familiar enough with the Cept or Gherkin formats I may try to extend this implementation to them as well if it hasn't been done already by then.